packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -226,14 +226,9 @@ class FlatTableState<T> extends State<FlatTable<T>> with AutoDisposeMixin {
     FlatTable<T> oldWidget,
     FlatTable<T> newWidget,
   ) {
-    final columnsChanged = !collectionEquals(
-          oldWidget.columns.map((c) => c.title),
-          newWidget.columns.map((c) => c.title),
-        ) ||
-        !collectionEquals(
-          oldWidget.columnGroups?.map((c) => c.title),
-          newWidget.columnGroups?.map((c) => c.title),
-        );
+    final columnsChanged =
+        !collectionEquals(oldWidget.columns, newWidget.columns) ||
+            !collectionEquals(oldWidget.columnGroups, newWidget.columnGroups);
```

Comment:
this may have unanticipated implications if columns and column group objects are accidentally being re-created. Then the instances would not match and this method would return true when we'd expect it to return false

Reply:
Hm, I can't remember exactly why I had needed this change and things seem to work properly without it, so I'll just revert it.